notebooks/Analyse-picks.ipynb

```diff
@@ -27,16 +30,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "particle_file = 'P15_J35_passthrough_particles.cs'"
+    "particle_file = '/Users/Expo/Desktop/Cryo-EM/Project/cryoem-analysis-tools/data/P15_J35_passthrough_particles.cs'"
```

Comment:
let's define a variable for the path.

Actually github is not meant to store data. Let's create a google drive repository and find a method to download the data from there. Let me know if you need help.

Reply:
Hi Fred I need your help with this. I tried to follow an online tutorial to set up google cloud for this but it's just too complicated. 
https://medium.com/@umdfirecoml/a-step-by-step-guide-on-how-to-download-your-google-drive-data-to-your-jupyter-notebook-using-the-52f4ce63c66c

Reply:
yes you're right, this might be too complicated for our needs. This was to cope with the fact that you do not have access to the `C092` directory yet...

Might be best to wait for you to have access. Then I will show you how to run the notebook from SLAC so this path will point to the data on the cluster. We can them delete the `.cs` file on GitHub and only keep the code.

Reply:
I'll go ahead and merge then